README.md

```diff
@@ -17,7 +17,7 @@
 
 | バージョン | 最新リリース                                                                      | リリース日 | End of Life |
 |------------|-----------------------------------------------------------------------------------|------------|-------------|
-|        3.9 |                                                                                   | Not yet    |             |
+|        3.9 | [3.9.13](https://github.com/kai2nenobu/win-python-installer/releases/tag/v3.9.13) | 2022-05-17 |             |
```

Comment:
Version 3.9.13 is not a security fix release.

[![image](https://user-images.githubusercontent.com/810042/175862546-1d4835ab-3177-4fc5-87b9-55862d25c5d6.png)](https://peps.python.org/pep-0596/#source-only-security-fix-releases)